Paintroid/src/main/java/org/catrobat/paintroid/dialog/SaveBeforeFinishDialog.java

```diff
@@ -72,7 +72,7 @@ public void onClick(DialogInterface dialog, int id) {
 	}
 
 	public enum SaveBeforeFinishDialogType {
-		BACK_TO_POCKET_CODE(R.string.closing_catroid_security_question_title, R.string.closing_security_question),
```

Comment:
This shouldn't be changed. The text should be "Back to project" but for some reason the string is missing in the values/string.xml [it is present in some translation files]